decentraland/unity-explorer
# feat: main app bootstrap

Implements a real application bootstrap by loading initial dependencies, realm, authentication, profile, player, world and finally teleports to spawn point.

# How to test?

1. The app should start with an icon animation, like a splash
2. The authentication should be shown allowing you to login as expected
3. Loading screen should appear
4. When it finishes, you should enter the world with your profile's avatar
5. Teleport to different spawn points, check that sdk7 scenes should load as expected
